main.js

```diff
@@ -28,7 +28,345 @@ function disapearEcsAndResize(event) {
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+        'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+      ],
+      [
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+        'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+        'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+      ],
+    ],
+  },
+  tags: {
+    m_d_popup: [
+      // [mobile, mobilePopup, desktop, desktopPopup]
+      [
+        ['css', 'html', 'bootstrap', 'Ruby'], ['html', 'bootstrap', 'Ruby on rails'],
+        ['css', 'html', 'bootstrap', 'Ruby'], ['html', 'bootstrap', 'Ruby on rails'],
+      ],
+      // [diferent order]
+      ['html', 'bootstrap', 'Ruby'],
+      ['html', 'bootstrap', 'Ruby'],
+      ['html', 'bootstrap', 'Ruby'],
+      ['html', 'bootstrap', 'Ruby'],
+      ['html', 'bootstrap', 'Ruby'],
+      ['html', 'bootstrap', 'Ruby'],
+    ],
+  },
+  buttons: {
+    m_d_popup: {
+      // [diferent order]
+      text: ['See Project', 'See Live', 'See Source'],
+      icon: [
+        'images/Icons/see_live_icon.svg',
+        'images/Icons/see_source_icon.svg',
+      ],
+      links: [
+        'https://github.com/Jord4N-AC/portfolio',
+      ],
+    },
+  },
+};
```

Comment:
- [ ] The pop-up works as expected. great job 👍 . I would recommend changing the structure of your array by using objects to separate the projects like the following example:
```js
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    name: "Second project",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
By implementing this structure, you will be able to add your future projects easily without any confusion.

Reply:
I don't really understand this

Reply:
not resolved

Reply:
You mean to my new changes?

Reply:
- You are supposed to place all the project info inside a single object, if you have 7 projects, you should have 7 objects inside your array but your objects have been grouped according to heading, description, and so on.
- Your project
![Annotation 2022-03-05 065544](https://user-images.githubusercontent.com/85113567/156870434-3b517cbe-5031-4430-ac50-5626e8edc793.jpg)